probatus/feature_elimination/feature_elimination.py

```diff
@@ -354,14 +370,23 @@ def fit(self, X, y, column_names=None):
         if self.random_state is not None:
             np.random.seed(self.random_state)
 
+        if columns_to_keep is None:
```

Comment:
If you set it to an empty list the check that you do earlier is always going to be True:
```        if columns_to_keep is not None:
```

Reply:
Yes, however if the list is empty,then there would be no elimination in the _get_current_features_to_remove method.

Reply:
I would propose not setting it as empty list and then you can make only the checks if it is None.

Reply:
Ok,then it makes sense to be consistent throughout. I will make the change.